### DISCUSSION TITLE
Lost profile and now can't find option

### DISCUSSION THREAD

**Discussion by Rigido71** [microsoft/terminal#16431] [Q&A]
Hi all,
some days ago I rebooted the PC (Win10 22H2) and Windows Terminal Preview (1.19.3172.0) complained about a missing default profile (more or less, I didn't get a screenshot of the error) and it started with some new settings, my color theme was there but font settings were reset, as well as mouse right click and some other key binding.
Now I'm trying to set the Appearance and I can clearly remember that there was a setting to dynamically change line spacing/height so window is always full and last line always "touches" bottom of the window,  is it still there somewhere? Fixed "Line Height" setting is 1.2 and grayed-out.
Now I would really love an easy way to backup profile/settings 😄

**Comment by lhecker** [comment]
> some days ago I rebooted the PC (Win10 22H2) and Windows Terminal Preview (1.19.3172.0) complained about a missing default profile 

If that happened, your disk may be failing, or it was too full to write anything on it. I'm sorry that this happened to you though. You can always find your settings in either of these places depending on the version that you're using:
```
C:\Users\lhecker\AppData\Local\Packages\Microsoft.WindowsTerminal_8wekyb3d8bbwe\LocalState
C:\Users\lhecker\AppData\Local\Packages\Microsoft.WindowsTerminalPreview_8wekyb3d8bbwe\LocalState
```

> Now I'm trying to set the Appearance and I can clearly remember that there was a setting to dynamically change line spacing/height so window is always full and last line always "touches" bottom of the window, is it still there somewhere? Fixed "Line Height" setting is 1.2 and grayed-out.

I'm not sure what you're asking for. We never had a setting that changes the line height dynamically depending on the terminal contents, if that's what you meant. As you've already found, the line height setting is here:
![image](https://github.com/microsoft/terminal/assets/2256941/1c1328d6-48ce-4d98-8cb3-6c3847520434)

It's not grayed out, it's simply empty and "1.2" is the placeholder text. You can click into the box and type a different line height into it.

**Reply by Rigido71** [reply]
Thank you Leonard,
maybe my memory isn't that good but is it possible that I read on a release note that this would never happen again and WT was able to dynamically adjust "content" to "fill the gap" with the bottom border?:

![image](https://github.com/microsoft/terminal/assets/29772377/1aedad7e-0b45-4854-aa0c-f34b1b47d300)

Thank you for the "Fixed line height" explanation but there is a small bug as place-holder should be shown with right cultural convention. Here in Italy we use comma as decimal separator but WT panel shows a greyed-out 1.2:
![image](https://github.com/microsoft/terminal/assets/29772377/ac31c52e-ec4e-45e7-b928-c7bb49438bbb)
but it will accept just comma separated values:
![image](https://github.com/microsoft/terminal/assets/29772377/b0410ec9-d57d-4ad5-950b-daa8172e9604)

Thanks again for your answers.
